modules/dspm-environments/main.tf

```diff
@@ -1,6 +1,10 @@
 data "aws_region" "current" {}
 
-resource "aws_vpc" "vpc" {
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+resource "aws_vpc" "VPC" {
```

Comment:
the resource name needs to be lowercased to prevent linting errors and so the reference to this vpc do not error